resources/management/commands/assign_area_regions.py

```diff
@@ -0,0 +1,114 @@
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+
+from resources.enums import AreaRegion
+from resources.models import Harbor, WinterStorageArea
+
+HARBOR_REGIONS = {
+    AreaRegion.EAST: [
+        "Airorannan venesatama",
+        "Aurinkolahden venesatama (Aurinkoranta)",
+        "Aurinkolahden venesatama (Pursilahdenranta)",
+        "Aurinkolahden venesatama (Pursilahdenranta)",
+        "Honkaluoto / Venesatama",
+        "Hopeasalmen venesatama",
+        "Kipparlahden venesatama",
+        "Koivuniemen venesatama",
+        "Laivalahden venesatama",
+        "Meri-Rastilan venesatama",
+        "Mustikkamaan venesatama",
+        "Nandelstadhin venesatama",
+        "Naurissalmen venesatama (Kipparlahdensilmukka 9)",
+        "Pikku Kallahden venesatama",
+        "Porslahden venesatama",
+        "Puotilan venesatama",
+        "Sarvaston venesatama",
+        "Strömsinlahden venesatama",
+        "Vasikkasaaren venesatama",
+        "Vuosaarenlahden venesatama",
+        "Vähäniityn venesatama",
+    ],
+    AreaRegion.WEST: [
+        "Eläintarhanlahden venesatama (Pitkänsillanranta)",
+        "Eläintarhanlahden venesatama",
+        "Hietalahdenallas / Venesatama",
+        "Katajanokan venesatama",
+        "Lähteelä retkisatama",
+        "Merihaan venesatama (Hakaniemenranta 13)",
+        "Merisatama (Ehrenströmintie 21)",
+        "Merisatama (Merisatama 28)",
+        "Merisatama (Merisatamanranta 20)",
+        "Pajalahden venesatama (Meripuistotie) / Venesatama",
+        "Pohjoisranta 3 / Venesatama",
+        "Ramsaynrannan venesatama (Ramsaynranta 4)",
+        "Ruoholahden venesatama (Jaalaranta)",
+        "Ruoholahden venesatama (Kellosaarenranta)",
+        "Ruoholahden venesatama (Ruoholahden kanava)",
+        "Salmisaaren venesatama",
+        "Saukonpaaden venesatama",
+        "Saunalahden venesatama (Ramsaynranta 6)",
+        "Siltavuoren venesatama",
+        "Tammasaaren allas",
+        "Tervasaari (Tervasaarenkannas 1) / Venesatama",
+    ],
+}
+
+WINTER_STORAGE_AREA_REGIONS = {
+    AreaRegion.EAST: [
+        "Marjaniemi",
+        "Puotila",
+        "Ruusuniemi I",
+        "Ruusuniemi II",
+        "Strömsinlahti",
+    ],
+    AreaRegion.WEST: ["Kaisaniemi", "Hernesaari", "Lähteelä", "Pajalahti"],
```

Comment:
the following marked WS areas are missing:
Rajasaari
Iso-Sarvasto
Porslahti
Laivalahti

Reply:
Rajasaari is neither on the excel sheet or the payment docs, but based on the map I'd say it's west? Or should it be blank?

Reply:
let's make it west